crates/rerun_c/src/rerun.h

```diff
@@ -345,13 +345,35 @@ extern void rr_recording_stream_set_time_nanos(
 
 /// Stops logging to the specified timeline for subsequent log calls.
 ///
-/// The timeline is still there, but it will not be updated with any new data.
+/// Clears out _both sequential and temporal_ timelines of the specified name.
+/// Refer to `disable_timeline_sequential`/`disable_timeline_temporal` if you need
+/// more fine-grained control.
 ///
-/// No-op if the timeline doesn't exist.
+/// The timelines are still there, but will not be updated with any new data.
+///
+/// No-op if the timelines don't exist.
 void rr_recording_stream_disable_timeline(
     rr_recording_stream stream, rr_string timeline_name, rr_error* error
 );
 
+/// Stops logging to the specified sequential timeline for subsequent log calls.
+///
+/// The timeline is still there, but will not be updated with any new data.
+///
+/// No-op if the timeline doesn't exist.
+void rr_recording_stream_disable_timeline_sequential(
+    rr_recording_stream stream, rr_string timeline_name, rr_error* error
+);
+
+/// Stops logging to the specified temporal timeline for subsequent log calls.
+///
+/// The timeline is still there, but will not be updated with any new data.
+///
+/// No-op if the timeline doesn't exist.
+void rr_recording_stream_disable_timeline_temporal(
```

Comment:
again, I don't think we shouldn't expose this to the user. A single disable should be enough.